internal/pkg/plugininstaller/jenkins/state.go

```diff
@@ -0,0 +1,34 @@
+package jenkins
+
+import (
+	"context"
+
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller"
+	"github.com/devstream-io/devstream/internal/pkg/statemanager"
+	"github.com/devstream-io/devstream/pkg/util/jenkins"
+)
+
+func GetStatus(options plugininstaller.RawOptions) (statemanager.ResourceState, error) {
+	opts, err := newJobOptions(options)
+	if err != nil {
+		return nil, err
+	}
+
+	client, err := jenkins.NewClient(opts.JenkinsURL, opts.BasicAuth)
+	if err != nil {
+		return nil, err
+	}
+	res := make(map[string]interface{})
```

Comment:
Maybe it's better to use statemanager.ResourceState instead of map[string]interface{} here.